pl_bolts/models/self_supervised/fixmatch/datasets.py

```diff
@@ -0,0 +1,187 @@
+import math
+
+import numpy as np
+from pytorch_lightning import LightningDataModule
+from torch.utils.data import DataLoader, SequentialSampler
+
+from pl_bolts.models.self_supervised.fixmatch.transforms import RandAugmentMC
+from pl_bolts.transforms.dataset_normalizations import cifar10_normalization, cifar100_normalization
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _TORCHVISION_AVAILABLE:
+    from torchvision import datasets, transforms
+else:  # pragma: no cover
+    warn_missing_pkg("torchvision")
+
+TRANS_WEAK = transforms.Compose(
+    [
+        transforms.RandomHorizontalFlip(),
+        transforms.RandomCrop(32, padding=int(32 * 0.125), padding_mode="reflect"),
+    ]
+)
+
+TRANS_STRONG = transforms.Compose(
+    [
+        transforms.RandomHorizontalFlip(),
+        transforms.RandomCrop(32, padding=int(32 * 0.125), padding_mode="reflect"),
+        RandAugmentMC(n=2, m=10),
+    ]
+)
+TRANS_STRONG_ANOTHER = transforms.Compose(
+    [
+        transforms.RandomResizedCrop(32, scale=(0.2, 1.0)),
+        transforms.RandomHorizontalFlip(p=0.5),
+        transforms.RandomApply([transforms.ColorJitter(0.4, 0.4, 0.4, 0.1)], p=0.8),
+        transforms.RandomGrayscale(p=0.2),
+    ]
+)
+
+
+class TransformSSL:
+    def __init__(self, dataset, mode="fixmatch"):
```

Comment:
```suggestion

    DATASET_NORMS = {
        "cifar10": cifar10_normalization,
        "cifar100": cifar100_normalization,
    }

    def __init__(self, dataset, mode="fixmatch"):
```
lets rather define as look-up-table